src/Tasks/CreateVisualBasicManifestResourceName.cs

```diff
@@ -37,7 +37,8 @@ protected override string CreateManifestName
         )
         {
             string culture = null;
-            if (fileName != null && itemSpecToTaskitem.TryGetValue(fileName, out ITaskItem item))
+            ITaskItem item = null;
+            if (fileName != null && itemSpecToTaskitem.TryGetValue(fileName, out item))
```

Comment:
Why?

Reply:
No longer needed as I'm not passing the item itself.